app/views/brexit_landing_page/show_A.html.erb

```diff
@@ -1,6 +1,6 @@
 <% content_for :page_class, "taxon-page taxon-page--grid" %>
 <% content_for :extra_headers do %>
-  <link rel="preload" as="image" href="<%= asset_path 'chevron-extend-hover.svg' %>">
+  <% @page_variant.analytics_meta_tag.html_safe %>
```

Comment:
Does this need to be a `<%= `, and if I'm right on that should a test be failing?

Reply:
Yup it does my bad.

`extra_headers` is actually a confusing name for what this content_for block is doing as it's actually only setting a meta tag and not a header here. I've renamed this to `meta_section`.

No test should be failing as there wasn't any checking the meta tag, the only test present was one checking the request header coming in. I've since amended the tests to check for the response headers and that the meta tag is being set.